kratos/python_scripts/analysis_stage.py

```diff
@@ -105,6 +105,7 @@ def Initialize(self):
             self.time = self._GetSolver().GetComputingModelPart().ProcessInfo[KratosMultiphysics.TIME]
         else:
             self.time = self.project_parameters["problem_data"]["start_time"].GetDouble()
+            self._GetSolver().GetComputingModelPart().ProcessInfo[KratosMultiphysics.TIME] = self.time

```

Comment:
OK. I wonder if we should set it not only in the computing model part but in the root.

Reply:
Good question, I set it only in the computing model part for two reasons:

- symmetry with the previous statement:
https://github.com/KratosMultiphysics/Kratos/blob/069695bd5962bbce9ecd480345a8c428765879e0/kratos/python_scripts/analysis_stage.py#L105

- The solver knows how to manage the time (calling the root model part would be going beyond the solver): https://github.com/KratosMultiphysics/Kratos/blob/069695bd5962bbce9ecd480345a8c428765879e0/kratos/python_scripts/analysis_stage.py#L62

but I haven't discussed it. BTW, since this PR adds a new test, it can be used in further discussions (root model part, filling buffer, initial conditions, etc)